### PR TITLE
HMAN-560:  logger info statements when hearing is updated to Exception

### PR DESCRIPTION
**Pull request by padmapriyanalam**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/HMAN-560

### Change description ###

Add extra logger info statements when hearing is updated to Exception

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by lang-ben — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/hmc/service/InboundQueueServiceImpl.java [hunk 1]
```diff
@@ -139,7 +139,8 @@ private void updateHearingAndStatus(Long hearingId, ErrorDetails errorDetails) {
                 .sendMessage(objectMapperService.convertObjectToJsonNode(hmcHearingResponse).toString(),
                              hmcHearingResponse.getHmctsServiceCode(),hearingId.toString());
             if (hmcHearingResponse.getHearingUpdate().getHmcStatus().equals(HearingStatus.EXCEPTION.name())) {
-                log.error("Hearing id: " +  hearingId + " updated to status Exception");
+                log.info("Hearing id: " + hearingId + "has response of type :" + MessageType.ERROR);
```
Comment (rajaram-nagarajan) [inline src/main/java/uk/gov/hmcts/reform/hmc/service/InboundQueueServiceImpl.java:142]: Let's combine both the logs into one error log
Ex: log.error("Hearing id: " + hearingId + " has response of type : " + MessageType.ERROR + " updated to status Exception");

Comment (RebeccaBaker) [reply src/main/java/uk/gov/hmcts/reform/hmc/service/InboundQueueServiceImpl.java:142]: Sorry Rajaram but we can't do that -  `log.error("Hearing id: " +  hearingId + " updated to status Exception");` is picked up by dynatrace to raise an incident ticket. Combining both statements will break the regex they look for

Comment (padmapriyanalam) [reply src/main/java/uk/gov/hmcts/reform/hmc/service/InboundQueueServiceImpl.java:142]: @RebeccaBaker,  Rajaram added comments in  the Jira https://tools.hmcts.net/jira/browse/HMAN-560 to address the dynatrace reg ex

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

File: src/integrationTest/java/uk/gov/hmcts/reform/hmc/config/MessageProcessorIT.java [hunk 2]
```diff
@@ -291,7 +294,8 @@ void shouldThrowMismatchOnRequestVersion() {
         assertEquals("Message of type HEARING_RESPONSE received", logsList.get(0).getMessage());
         assertEquals("Error processing message with Hearing id 2000000000 exception was "
                          + "Cannot find request version 10 for hearing 2000000000", logsList.get(1).getMessage());
-        assertEquals("Hearing id: 2000000000 updated to status Exception", logsList.get(2).getMessage());
+        assertEquals("Hearing id: 2000000000 Case ref: 9372710950276233 Service Code: TEST"
```
Comment (markdathornehmcts) [inline src/integrationTest/java/uk/gov/hmcts/reform/hmc/config/MessageProcessorIT.java:297]: Just a note to start using getFormattedMessage ...for when messages are updated to use variables for formatting the message

File: src/main/java/uk/gov/hmcts/reform/hmc/service/InboundQueueServiceImpl.java [hunk 3]
```diff
@@ -106,7 +110,9 @@ public void catchExceptionAndUpdateHearing(Map<String, Object> applicationProper
                 hearingEntity.setStatus(EXCEPTION.name());
                 hearingEntity.setErrorDescription(exception.getMessage());
                 hearingRepository.save(hearingEntity);
-                log.error("Hearing id: " +  hearingId + " updated to status Exception");
+                log.error(HEARING_ID_FOR_LOG + hearingId + CASE_REF + hearingEntity.getLatestCaseReferenceNumber()
```
Comment (markdathornehmcts) [inline src/main/java/uk/gov/hmcts/reform/hmc/service/InboundQueueServiceImpl.java:113]: Could use message formatting with variables

File: src/test/java/uk/gov/hmcts/reform/hmc/service/InboundQueueServiceTest.java [hunk 4]
```diff
@@ -1018,11 +1023,21 @@ private ListAppender<ILoggingEvent> setupLogger() {
...
-    private void assertDynatraceLogMessage(ListAppender<ILoggingEvent> listAppender, String hearingID) {
+    private void assertDynatraceLogMessage(ListAppender<ILoggingEvent> listAppender, String hearingID,
+                                           String caseRef, String serviceCode, MessageType messageType) {
         List<ILoggingEvent> logsList = listAppender.list;
         int finalErrorIndex = logsList.size() - 1;
         assertEquals(Level.ERROR, logsList.get(finalErrorIndex).getLevel());
-        assertEquals("Hearing id: " + hearingID + " updated to status Exception",
-                     logsList.get(finalErrorIndex).getMessage());
+        if (null != messageType) {
+            assertEquals("Hearing id: " + hearingID + " Case ref: " + caseRef
```
Comment (markdathornehmcts) [inline src/test/java/uk/gov/hmcts/reform/hmc/service/InboundQueueServiceTest.java:1032]: format message

**Comment by padmapriyanalam** [comment]
Create a new PR [Hmc-cft-hearing-service](https://github.com/hmcts/hmc-cft-hearing-service/pull/521)
Closing this

**Pull request closed without merging**